i18n/xgettext-go/main.go

```diff
@@ -192,6 +192,14 @@ var formatTime = func() string {
 	return time.Now().Format("2006-01-02 15:04-0700")
 }
 
+func mustFprintf(w io.Writer, format string, a ...interface{}) (n int, err error) {
+	n, err = fmt.Fprintf(w, format, a...)
+	if err != nil {
+		panic(fmt.Sprintf("cannot write output: %v", err))
```

Comment:
What is the point of the error return value if it is always nil?

Reply:
Good point, happy to remove it.